certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,707 @@
+                          "type": "string",
+                          "description": "CA_POSTCODE"
+                        }
+                      },
+                      "description": "CA_CUSTOMER_REQUEST"
+                    },
+                    "CA_POLICY_REQUEST": {
+                      "type": "object",
+                      "properties": {
+                        "CA_POLICY_SPECIFIC": {
+                          "type": "string",
+                          "description": "CA_POLICY_SPECIFIC"
+                        },
+                        "CA_POLICY_NUM": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "CA_POLICY_NUM"
+                        },
+                        "CA_POLICY_COMMON": {
+                          "type": "object",
+                          "properties": {
+                            "CA_ISSUE_DATE": {
+                              "type": "string",
+                              "description": "CA_ISSUE_DATE"
+                            },
+                            "CA_PAYMENT": {
+                              "type": "integer",
+                              "format": "int32",
+                              "description": "CA_PAYMENT"
+                            },
+                            "CA_BROKERID": {
+                              "type": "integer",
+                              "format": "int32",
+                              "description": "CA_BROKERID"
+                            },
+                            "CA_EXPIRY_DATE": {
+                              "type": "string",
+                              "description": "CA_EXPIRY_DATE"
+                            },
+                            "CA_LASTCHANGED": {
+                              "type": "string",
+                              "description": "CA_LASTCHANGED"
+                            },
+                            "CA_BROKERSREF": {
+                              "type": "string",
+                              "description": "CA_BROKERSREF"
+                            }
+                          },
+                          "description": "CA_POLICY_COMMON"
+                        }
+                      },
+                      "description": "CA_POLICY_REQUEST"
+                    },
+                    "CA_RETURN_CODE": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "CA_RETURN_CODE"
+                    },
+                    "CA_CUSTOMER_NUM": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "CA_CUSTOMER_NUM"
+                    }
+                  },
+                  "description": "LGCMAREA"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get customer details"
```

Comment:
@RTkyndryl The 'description' property is required. please add description under the summary. 
```
"summary": "Get customer details",
"description": "Your description"
```